src/lib/userStorage/UserStorageClass.js

```diff
@@ -1072,8 +1076,10 @@ export class UserStorage {
     return this.profileStorage.getPublicProfile(key, string)
   }
 
-  getFaceIdentifier(): string {
-    return this.wallet.getAccountForType('faceVerification').replace('0x', '')
+  async getFaceIdentifiers(): string {
+    const v2Identifier = await this.wallet.sign(FV_IDENTIFIER_MSG2({ account: this.wallet.account }), 'gd')
```

Comment:
nice, I've re-thought the suggestions been done and just wanted to propose such approach because both ids are always used together